src/Components/Input/Input.jsx

```diff
@@ -0,0 +1,5 @@
+import "./input.scss";

```

Comment:
Change the file name from `src/Components/Input/Input.jsx` to `src/Components/Input/input.jsx`